model-optimizer/mo/utils/ir_reader/extenders/priorbox_extender.py

```diff
@@ -17,7 +17,8 @@ def extend(op: Node):
         for attr in attrs:
             PriorBox_extender.attr_restore(op, attr)
 
-        if op.graph.graph['cmd_params'].framework == 'mxnet':
+        if 'framework' in op.graph.graph['cmd_params'] and op.graph.graph['cmd_params'].framework == 'mxnet':
```

Comment:
can we have a case when `framework` not in `op.graph.graph['cmd_params']`? Is it an example of IR creation using nGraph API? If yes, is it agreed not to have this attribute?